configuration/configuration.go

```diff
@@ -48,6 +48,11 @@ type Configuration struct {
 	// deprecated. Please use Log.Level in the future.
 	Loglevel Loglevel `yaml:"loglevel,omitempty"`
 
+	// DriverPlugins is a path where driver plugins are expected to be found
+	DriverPlugins *string `yaml:"driverplugins,omitempty"`
```

Comment:
So, I took a closer look into this (via containerd) and it seems like they should work like plugins should be a list of paths to plugins or plugin directories:

```
plugins:
  - /plugins/
  - /plugin/foo.so
```

We don't actually need to differentiate on type if we just have the plugins register themselves in their `init` function. This way, they will just show up in the driver/auth list like any other plugin.

I am not sure that this model works in full, but this will make it a lot easier to setup plugins in practice.